virtme/guest/virtme-sshd-script

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+#
+# Initialize ssh server for remote connections (option `--server ssh`)
+
+HOME=$(getent passwd "${virtme_user}" | cut -d: -f6)
+cat ${HOME}/.ssh/id_rsa.pub >> ${HOME}/.ssh/authorized_keys
```

Comment:
Maybe safer to cat all `id_*.pub` files? e.g. what if the default key is a `id_ed25519` (is it not the new default algo?)